analysis/classification/main.py

```diff
@@ -116,12 +116,16 @@ def logout():
 if operation == 'Assign Data':
     if 'sampling_mode' not in st.session_state:
         st.session_state['sampling_mode'] = ''
+    if 'target_probability' not in st.session_state:
+        st.session_state['target_probability'] = 0.5
```

Comment:
Not critical, but could consider ensuring that if active learning is turned off, the probability is set to a value that makes clearer that it's not on.

Reply:
It will be set to None now